<!DOCTYPE html>
<html lang="pt-br">

<head>
  <title>Manojkumar</title>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <!-- Template css Files -->
  <link rel="stylesheet" href="css/style.css" type="text/css">

  <!-- Live Style Switcher - Demo Only  -->
  <link rel="stylesheet" class="alternate-style" title="pink" href="css/skins/pink.css" type="text/css" >
  <link rel="stylesheet" class="alternate-style" title="blue" href="css/skins/blue.css" type="text/css" disabled>
  <link rel="stylesheet" class="alternate-style" title="green" href="css/skins/green.css" type="text/css" disabled>
  <link rel="stylesheet" class="alternate-style" title="orange" href="css/skins/orange.css" type="text/css" disabled>
  <link rel="stylesheet" class="alternate-style" title="yellow" href="css/skins/yellow.css" type="text/css" disabled>
  <link rel="stylesheet" href="css/styleSwitcher.css" type="text/css">
</head>

<body class="light">

  <!-- Preloader -->
  <div class="preloader">
    <div class="loader">

    </div>
  </div>
  <!-- Preloader End -->

  <!-- Main Container -->
  <div class="main-container">
    <!-- Aside -->
    <div class="aside">
      <!-- Nav Toggler Btn -->
      <div class="nav-toggler">
        <span></span>
      </div>
      <div class="aside-inner">
        <!-- Logo -->
        <div class="logo">
          <a href="#">Manoj</a>
        </div>
        <!-- Nav -->
        <ul class="nav">
          <li><a href="#home" class="active"><i class="fa fa-home"></i> Home</a></li>
          <li><a href="#about"><i class="fa fa-user"></i> About</a></li>
          <li><a href="#publications"><i class="fa fa-list"></i> Publications</a></li>
          <li><a href="#service"><i class="fa fa-briefcase"></i> Services</a></li>
          <li><a href="#blog"><i class="fa fa-envelope"></i> Blog</a></li>
          <li><a href="#testimonial"><i class="fa fa-quote-left"></i> Testimonials</a></li>
          <li><a href="#contact"><i class="fa fa-comments"></i> Contact</a></li>
        </ul>
        <!-- Copyright -->
        <div class="copyright-text">
          &copy; 2025 manojkumar kumaran
        </div>
      </div>
    </div>
    <!-- Aside End -->
    <!-- Main Content -->
    <div class="main-content">

      <!-- Home Section -->
      <section class="home section active" id="home">
        <div class="container">
          <div class="intro">
            <img src="images/manoj_1.jpg" alt="profile" class="shadow-dark-1">
            <h1>Manojkumar Kumaran</h1>
            <span class="iTyped"></span>
            <div class="social-links">
              <a href="https://x.com/manoj290991" target="_blank"><i class="fa fa-twitter"></i></a>
              <a href="https://www.facebook.com/k.manojkumar2991" target="_blank"><i class="fa fa-facebook"></i></a>
              <!-- <a href="https://www.instagram.com/k.manojkumar2991/" target="_blank"><i class="fa fa-instagram"></i></a> -->
              <a href="https://www.linkedin.com/in/manojkumar-kumaran-57991543/" target="_blank"><i class="fa fa-linkedin"></i></a>
              <a href="https://github.com/mano2991" target="_blank"><i class="fa fa-github"></i></a>
              <a href="https://www.researchgate.net/profile/Manoj-Kumar-103?ev=hdr_xprf" target="_blank" rel="noopener noreferrer"><i class="fa fa-graduation-cap"></i></a>
            </div>
          </div>
        </div>
      </section>
      <!-- Home Section End -->

      <!-- About Section -->
      <section class="about section" id="about">
        <div class="container">
          <div class="row">
            <div class="section-title padd-15">
              <h2>About Me</h2>
            </div>
          </div>
          <div class="row">
            <div class="about-content padd-15">
              <div class="row">
                <div class="about-text padd-15">
                  <h3>Hi! I'm Manojkumar Kumaran and <span>Bioinformatician</span></h3>
                  <p style="text-align: justify;">
                      I am a Computational Neurobiologist and Software Developer passionate about understanding the molecular and circuit mechanisms underlying neurological disorders and regeneration. 
                      I specialize in single-cell multi-omics, Hi-C, ChIP-seq, and ATAC-seq analysis, integrating computational and experimental approaches to study axon regeneration, neurodegeneration, and brain plasticity. 
                      My research focuses on how genomic architecture, transcription factors, and chromatin accessibility shape neuronal function in health and disease.
                      Beyond research, I have extensive experience in software development, building automated bioinformatics pipelines, and applying machine learning for large-scale genomic analysis.
                      I thrive on solving complex biological problems using computational tools and data-driven insights.
                      In my free time, I enjoy wildlife photography, gaming, and cooking.
                  </p>
                </div>
              </div>
              <div class="row">
                <div class="personal-info padd-15">
                  <div class="row">
                    <div class="info-item padd-15">
                      <p>Birthday : <span>29 September 1991</span></p>
                    </div>
                    <div class="info-item padd-15">
                      <p>Age : <span>33</span></p>
                    </div>
                      <!-- <div class="info-item padd-15">
                      <p>Website : <span>www.domain.com</span></p>
                    </div> -->
                    <div class="info-item padd-15">
                      <p>Email : <span>manojkumarbioinfo@gmail.com</span></p>
                    </div>
                    <div class="info-item padd-15">
                      <p>Degree : <span>PhD</span></p>
                    </div>
                    <div class="info-item padd-15">
                      <p>Phone : <span>+91 7708556754</span></p>
                    </div>
                    <div class="info-item padd-15">
                      <p>City : <span>Madurai</span></p>
                    </div>
                    <div class="info-item padd-15">
                      <p>Freelance : <span>Available</span></p>
                    </div>
                    <div class="info-item padd-15">
                      <p>Nationality : <span>Indian</span></p>
                  </div>
                  <div class="info-item padd-15">
                    <p>Languages : <span>English, Tamil</span></p>
                </div>
                <div class="info-item padd-15">
                  <p>Current Role : <span>Research Associate</span></p>
                </div>
                <div class="info-item padd-15">
                  <p>Research Interests : <span>Multi-Omics, AI in Neuroscience</span></p>
              </div>
              <div class="info-item padd-15">
                <p>Hobbies : <span>Wildlife Photography, Gaming, Cooking</span></p>
            </div>
            
                 </div>
                  <div class="row">
                    <div class="buttons padd-15">
                      <a href="Manoj_CV.pdf" target="_blank" class="btn">Download Cv</a>
                      <!-- <a href="#contact" data-section-index="1" class="btn hire-me">Hire me</a> -->
                    </div>
                  </div>
                </div>
                <div class="skills padd-15">
                  <div class="row">
                    <div class="skill-item padd-15">
                      <h5>Shell, Python, R, Docker </h5>
                      <div class="progress">
                        <div class="progress-in" style="width: 86%;"></div>
                        <div class="skill-porcent">
                          86%
                        </div>
                      </div>
                    </div>
                    <div class="skill-item padd-15">
                      <h5>Clinical WES/WGS Sequecning analysis</h5>
                      <div class="progress">
                        <div class="progress-in" style="width: 90%;"></div>
                        <div class="skill-porcent">
                          90%
                        </div>
                      </div>
                    </div>
                    <div class="skill-item padd-15">
                      <h5>Single Cell Genomics sequecning analysis</h5>
                      <div class="progress">
                        <div class="progress-in" style="width: 80%;"></div>
                        <div class="skill-porcent">
                          80%
                        </div>
                      </div>
                    </div>
                    <div class="skill-item padd-15">
                      <h5>Bulk ATAC, Chip, Cut&Run, HiC analysis</h5>
                      <div class="progress">
                        <div class="progress-in" style="width: 75%;"></div>
                        <div class="skill-porcent">
                          75%
                        </div>
                      </div>
                    </div>
                    <div class="skill-item padd-15">
                      <h5>Structural bioinformatics </h5>
                      <div class="progress">
                        <div class="progress-in" style="width: 85%;"></div>
                        <div class="skill-porcent">
                          85%
                        </div>
                      </div>
                     </div>
                     <div class="skill-item padd-15">
                      <h5>Machine Learning, Animal Behavior </h5>
                      <div class="progress">
                        <div class="progress-in" style="width: 75%;"></div>
                        <div class="skill-porcent">
                          75%
                        </div>
                      </div>
                    </div>
                    <div class="skill-item padd-15">
                      <h5>Java, Html, Bootstrap</h5>
                      <div class="progress">
                        <div class="progress-in" style="width: 60%;"></div>
                        <div class="skill-porcent">
                          60%
                        </div>
                      </div>
                    </div>
                    <div class="skill-item padd-15">
                      <h5>Server and Network Manager</h5>
                      <div class="progress">
                        <div class="progress-in" style="width: 80%;"></div>
                        <div class="skill-porcent">
                          80%
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>

              <div class="row">
                <div class="education padd-15">
                  <h3 class="title">Education</h3>
                  <div class="row">
                    <div class="timeline-box padd-15">
                      <div class="timeline shadow-dark">
                        <!-- timeline Item -->
                        <div class="timeline-item">
                          <div class="circle-dot"></div>
                          <h6 class="timeline-date"><i class="fa fa-calendar"></i> 2015 - 2021</h6>
                          <h4 class="timeline-title">PhD In bioinformatics, at Arvind Medical Research Foundation</h4>
                          <p class="timeline-text">
                            I developed an eye disease-specific variant prioritization tool, <b>eyeVarP</b>, which utilizes a random forest algorithm to analyze pathogenic variants associated with eye diseases and other disorders.  
                            Additionally, I created the <b>VarP</b> tool, a generalized pipeline designed to filter missense and insertion-deletion variants and predict their pathogenicity from exome or genome sequencing data.
                        </p>
                        </div>
                        <!-- timeline Item End -->
                        <!-- timeline Item -->
                        <div class="timeline-item">
                          <div class="circle-dot"></div>
                          <h6 class="timeline-date"><i class="fa fa-calendar"></i> 2009 - 2015</h6>
                          <h4 class="timeline-title">Master of Technology (M.Tech) in Bioinformatics</h4>
                          <p class="timeline-text">
                              I pursued a <b>six-year integrated M.Tech program in Bioinformatics</b> at Bharathidasan University, gaining expertise in computational biology, genomics, proteomics, and data-driven research methodologies.  
                              Throughout my academic journey, I developed a strong foundation in biological data analysis, machine learning applications in genomics, structural bioinformatics, and molecular modeling.  
                              I actively worked on research projects related to genome sequencing, variant analysis, and computational drug discovery, equipping myself with both wet-lab and computational skills essential for bioinformatics research.  
                          </p>
                        </div>
                        <!-- timeline Item End -->
                        <!-- timeline Item 
                        <div class="timeline-item">
                          <div class="circle-dot"></div>
                          <h6 class="timeline-date"><i class="fa fa-calendar"></i> 2008 - 2011</h6>
                          <h4 class="timeline-title">Bachelor Degree</h4>
                          <p class="timeline-text">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque scelerisque diam non
                            nisi semper, et elementum lorem ornare.</p>
                        </div>
                         timeline Item End -->
                      </div>
                    </div>
                  </div>
                </div>
                <div class="experience padd-15">
                  <h3 class="title">Experience</h3>
                  <div class="row">
                    <div class="timeline-box padd-15">
                        <div class="timeline shadow-dark">
                            <!-- Timeline Item -->
                            <div class="timeline-item">
                                <div class="circle-dot"></div>
                                <h6 class="timeline-date"><i class="fa fa-calendar"></i> 2022 - Present</h6>
                                <h4 class="timeline-title">Research Associate, CSIR-CCMB</h4>
                                <p class="timeline-text">
                                    Developed an automated pipeline for analyzing single-cell RNA-seq, single-cell ATAC-seq, CUT&RUN, Hi-C, and bulk ATAC-seq data.  
                                    Built a machine learning model for mouse behavioral analysis.  
                                    Maintained the lab website and the GitHub repository.
                                </p>
                            </div>
                            <!-- Timeline Item End -->
                
                            <!-- Timeline Item -->
                            <div class="timeline-item">
                                <div class="circle-dot"></div>
                                <h6 class="timeline-date"><i class="fa fa-calendar"></i> 2021 - 2022</h6>
                                <h4 class="timeline-title">Data Scientist, INDX Technologies</h4>
                                <p class="timeline-text">
                                    Developed an automated tool for analyzing Whole Exome Sequencing (WES) data from cancer patient datasets.  
                                    Created an automated annotation tool to classify genetic variants based on available drug information.
                                </p>
                            </div>
                            <!-- Timeline Item End -->
                
                            <!-- Timeline Item -->
                            <div class="timeline-item">
                                <div class="circle-dot"></div>
                                <h6 class="timeline-date"><i class="fa fa-calendar"></i> 2015 - 2021</h6>
                                <h4 class="timeline-title">Junior Research Fellow, Aravind Medical Research Foundation</h4>
                                <p class="timeline-text">
                                    Developed and benchmarked analytical tools and pipelines for Next-Generation Sequencing (NGS) data.  
                                    Applied biostatistics and bioinformatics algorithms to integrate multi-level biological data with functional genomics for hypothesis generation, variant identification, and variant prioritization.  
                                    Collaborated across multiple disciplines and contributed to various research projects.  
                                    Maintained the lab website and the GitHub repository.
                                </p>
                            </div>
                            <!-- Timeline Item End -->
                        </div>
                    </div>
                </div>
                
                </div>
              </div>

            </div>
          </div>
        </div>
      </section>
      <!-- About Section End -->

      <!-- publication Section -->
      <section class="publication section" id="publications">
        <div class="container">
          <div class="row">
            <div class="section-title padd-15">
              <h2>Publications</h2>
              <br>
              <br>
              <p style="color: var(--text-color)">Total Citations: 72 | H-index: 4 | Research Score: 69.5 | Reads: 3640</p>
            </div>
          </div>
          <div class="row">
            <div class="publications">
    <div class="publication-list">
        <!-- Paper 1 - 2025 -->
        <div class="publication-card">
          <h4><a href="https://www.jneurosci.org/content/early/2025/01/01/JNEUROSCI.1508-24.2024" target="_blank">
              Single-nuclei sequencing reveals a robust corticospinal response to nearby axotomy but overall insensitivity to spinal injury
          </a></h4>
          <p><strong>Manojkumar Kumaran et al.</strong> <i>Journal of Neuroscience</i> (2025)</p>
          <details>
              <summary>Show Abstract</summary>
              <p>The ability of neurons to sense and respond to damage is crucial for maintaining homeostasis and facilitating nervous system repair. For some cell types, notably dorsal root ganglia and retinal ganglion cells, extensive profiling has uncovered a significant transcriptional response to axon injury, which influences survival and regenerative outcomes. In contrast, the injury responses of most supraspinal cell types, which display limited regeneration after spinal damage, remain mostly unknown. In this study, we used single-nuclei sequencing in adult male and female mice to profile the transcriptional responses of diverse supraspinal cell types to spinal injury. Surprisingly, thoracic spinal injury induced only modest changes in gene expression across all populations, including corticospinal tract (CST) neurons. Additionally, CST neurons exhibited minimal response to cervical injury but showed a much stronger reaction to intracortical axotomy, with upregulation of numerous regeneration and apoptosis-related transcripts shared with injured DRG and RGC neurons. Thus, the muted response of CST neurons to spinal injury is linked to the injury's distal location, rather than intrinsic cellular characteristics. More broadly, these findings indicate that a central challenge for enhancing regeneration after a spinal injury is the limited detection of distant injuries and the subsequent modest baseline neuronal response.</p>
          </details>
        </div>

        <!-- Paper 2 - 2024 -->
        <div class="publication-card">
          <h4><a href="https://www.nature.com/articles/s10038-024-01267-0" target="_blank">
              Investigating druggable kinases for targeted therapy in cancer
          </a></h4>
          <p><strong>Kumar Jeyaprakash, Manojkumar Kumaran et al.</strong> <i>Journal of Human Genetics</i> (2024)</p>
          <details>
              <summary>Show Abstract</summary>
              <p>Retinoblastoma (RB) is a childhood retinal neoplasm and commonly treated with cytotoxic chemotherapeutic agents. However, these therapeutic approaches often lead to diverse adverse effects. A precise molecular therapy will alleviate these side effects and offer better treatment outcomes. Over the years, kinases have become potential drug targets in cancer therapy. Hence, we aimed to investigate genetic alterations of putative kinase drug targets in RB. Targeted exome sequencing was performed on 35 RB tumors with paired blood samples using a gene panel consisting of 29 FDA-approved kinase genes. Single nucleotide variants were analyzed for pathogenicity using an in-house pipeline and copy number variations (CNVs) were detected by a depth of coverage and CNVPanelizer. The correlation between genetic changes and clinicopathological features was assessed using GraphPad Prism. Three somatic mutations, two in ERBB4 and one in EGFR were identified. Two of these mutations (ERBB4 c.C3836A & EGFR c.A1196T) were not reported earlier. CNV analysis revealed recurrent gains of ALK, MAP2K2, SRC, STK11, and FGFR3 as well as frequent losses of ATM, PI3KCA and ERBB4. Notably, nonresponsive tumors had a higher incidence of amplifications in clinically actionable genes such as ALK. Moreover, ALK gain and ATM loss were strongly correlated with optic nerve head invasion. In conclusion, our study revealed genetic alterations of druggable kinases in RB, providing preliminary insights for the exploration of kinase-targeted therapy in RB.</p>
          </details>
        </div>

        <!-- Paper 3 - 2024 -->
        <div class="publication-card">
          <h4><a href="https://aacrjournals.org/cancerres/article/83/2_Supplement_2/PR007/712515/Abstract-PR007-Comprehensive-ctDNA-profiling" target="_blank">
              Comprehensive ctDNA profiling reveals potential resistance mechanisms
          </a></h4>
          <p><strong>Shafi, Gowhar , Manojkumar Kumaran et al.</strong> <i>Cancer Research</i> (2024)</p>
          <details>
              <summary>Show Abstract</summary>
              <p>Background: Genomic profiling has revolutionized precision oncology impacting the diagnosis, prognosis, and therapy decisions. Considering high spatiotemporal diversity and heterogenicity of breast tumor-cell genomes, small-gene panels often fail to capture rare but important genomic alterations. Conversely, comprehensive ctDNA sequencing approaches enable the identification of under characterized 'long tailed driver' genomic alterations and capture Intra and inter metastatic heterogeneity. Here, we demonstrate the clinical utility of comprehensive genome profiling with higher sensitivity to predict the possibility of metastasis in early-stage breast cancer patients. Methods: We retrospectively analyzed ctDNA and genomic DNA (gDNA) from FFPE samples as well as circulating tumor cells (CTC) in 10 treatment-naive hormone positive and HER2 negative, primary-stage breast cancer patients [GS1] using the OncoIndx comprehensive 600 gene panel. The panel captures all important cancer-relevant genomic alterations including Tumor Mutation Burden (TMB), Micro Satellite Instability (MSI), homologous recombination deficiency (HRD) prediction, and cfDNA tumor fraction (TF). CTCs were enumerated from 1.5 ml of blood using the OncoDiscover platform approved by the Drug Controller General of India having anti-EpCAM antibody-mediated immunomagnetic nanoparticles. CTCs were confirmed for cytokeratin 18+ and DAPI + markers and the absence of CD45. Results: Comprehensive genomic profile obtained from ctDNA and gDNA from FFPE of early-stage breast cancer patients predominantly exhibited the presence of alterations in PIK3CA and ESR1 signaling pathways. PIK3CAmutations were present in 77% and 44% of baseline ctDNA and gDNA samples, while ESR1 mutations were present in 44% and 22% of baseline ctDNA and gDNA, respectively. In addition, we observed about 70% additional driver mutations in ctDNA samples suggesting shedding of ctDNA together with CTC (80% positive), a likely positive biomarker of metastasis. About 50% of the patients showed higher TMB and HRD. Notably, TF representing ctDNA varied between 13% to 27% in blood samples with a corresponding ploidy range of 2.9 to 4.7. Surprisingly, ~50% of the patient population matched the mutation profile of clinically confirmed metastatic patients. All the patients harboring potential metastatic driver alterations showed the presence of CTCs in peripheral blood. Conclusions: Comprehensive ctDNA genomic profiling showed potential metastasis driving alterations suggesting the role of ctDNA-based liquid biopsy to predict metastasis in early breast cancer patients. We observed enhanced TF at the time of diagnosis, possibly due to the presence of distant metastasis, high disease burden, and aggressive tumor biology. Our results suggest that ctDNA dynamics at the time of disease presentation can predict early metastasis, and may demonstrate the divergent response of tumor heterogeneity to treatment in early-stage breast cancer.</p>
          </details>
        </div>

        <!-- Paper 4 - 2023 -->
        <div class="publication-card">
          <h4><a href="https://www.sciencedirect.com/science/article/pii/S1098360023008754" target="_blank">
              eyeVarP: a computational framework for the identification of pathogenic variants specific to eye disease
          </a></h4>
          <p><strong>Manojkumar Kumaran et al.</strong> <i>Genetics in Medicine</i> (2023)</p>
          <details>
              <summary>Show Abstract</summary>
              <p>Purpose: Disease-specific pathogenic variant prediction tools that differentiate pathogenic variants from benign have been improved through disease specificity recently. However, they have not been evaluated on disease-specific pathogenic variants compared with other diseases, which would help to prioritize disease-specific variants from several genes or novel genes. Thus, we hypothesize that features of pathogenic variants alone would provide a better model. Methods: We developed an eye disease–specific variant prioritization tool (eyeVarP), which applied the random forest algorithm to the data set of pathogenic variants of eye diseases and other diseases. We also developed the VarP tool and generalized pipeline to filter missense and insertion-deletion variants and predict their pathogenicity from exome or genome sequencing data, thus we provide a complete computational procedure. Results:eyeVarP outperformed pan disease–specific tools in identifying eye disease–specific pathogenic variants under the top 10. VarP outperformed 12 pathogenicity prediction tools with an accuracy of 95% in correctly identifying the pathogenicity of missense and insertion-deletion variants. The complete pipeline would help to develop disease-specific tools for other genetic disorders. Conclusion:eyeVarP performs better in identifying eye disease–specific pathogenic variants using pathogenic variant features and gene features. Implementing such complete computational procedure would significantly improve the clinical variant interpretation for specific diseases.</p>
          </details>
        </div>

        <!-- Paper 5 - 2021 -->
        <div class="publication-card">
          <h4><a href="https://pubmed.ncbi.nlm.nih.gov/34751623/" target="_blank">
              Clinical reassessments and whole-exome sequencing
          </a></h4>
          <p><strong>Manojkumar Kumaran et al.</strong> <i>Ophthalmic Genetics</i> (2021)</p>
          <details>
              <summary>Show Abstract</summary>
              <p>Background: The diagnosis of retinal dystrophies can be challenging due to the spectrum of protean phenotypic manifestations. This study employed trio-whole-exome sequencing (trio-WES) to unveil the genetic cause of an inherited retinal disorder in a south Indian family. Materials and methods: Proband's initial ophthalmic examinations was performed in the year 2016. WES was performed on a proband-parent trio to identify causative mutation followed by Sanger validation, segregation analysis, sequence and structure-based computational analysis to assess its pathogenicity. Based on the genetic findings, detailed clinical reassessments were performed in year 2020 for the proband and available family members. Results: WES revealed a novel homozygous BEST1 mutation c.G310A (p.D104N) in the proband and heterozygous for the parents, indicating autosomal recessive inheritance. Segregation analysis showed heterozygous mutation in maternal grandfather and normal genotype for younger brother and maternal grandmother. Moreover, the structure-based analysis revealed the mutation p.D104N in the cytoplasmic domain, causing structural hindrance by altering hydrogen bonds and destabilizing the BEST1 protein structure. Proband's clinical assessments were consistent with autosomal recessive bestrophinopathy (ARB) phenotype. Additionally, characteristic absent light rise and decreased light peak-to-dark trough ratio (LP:DT) was observed bilaterally in EOG. Conclusions: Our study demonstrates the utility of WES and clinical re-evaluations in establishing the precise diagnosis of autosomal recessive bestrophinopathy associated with a novel mutation, thus expanding the BEST1-related mutation spectrum.</p>
          </details>
        </div>

        <!-- Paper 6 - 2021 -->
        <div class="publication-card">
          <h4><a href="https://www.taylorfrancis.com/chapters/edit/10.1201/9781003005926-7/bioinformatics-whole-exome-studies-manojkumar-kumaran-bharanidharan-devarajan" target="_blank">
              Bioinformatics for Whole Exome Studies
          </a></h4>
          <p><strong>Manojkumar Kumaran et al.</strong> <i>Bioinformatics and Human Genomics Research</i> (2021)</p>
          <details>
              <summary>Show Abstract</summary>
              <p>Recent advances in next-generation sequencing (NGS) methods have greatly promoted the NGS-based genome/exome studies in discovering genetic variants for human diseases. Whole-genome sequencing (WGS) remains prohibitively expensive and requires concurrent development of bioinformatics approaches. Whole exome sequencing (WES), focusing on only the protein-coding sequence of the human genome, is now a widely used cost-effective approach that could identify disease-causing variants. Here, we provide the performance assessment of various aligners and variant calling tools using simulated and standard human exome datasets and highlight their combinations that improve the detection of single nucleotide variants (SNVs) and InDels. Further, we provide general recommendations for bioinformatic analysis for WES methods, focusing on variant filtering and prioritization for clinical use.</p>
          </details>
        </div>

        <!-- Paper 7 - 2021 -->
        <div class="publication-card">
          <h4><a href="https://pubmed.ncbi.nlm.nih.gov/34427245/" target="_blank">
              Whole-exome sequencing identifies multiple pathogenic variants
          </a></h4>
          <p><strong>Manojkumar Kumaran et al.</strong> <i>Indian Journal of Ophthalmology</i> (2021)</p>
          <details>
              <summary>Show Abstract</summary>
              <p>Purpose: To identify the pathogenic variants associated with primary open-angle glaucoma (POAG) using whole-exome sequencing (WES) data of a large South Indian family. Methods: We recruited a large five-generation South Indian family (n = 84) with a positive family history of POAG (n = 19). All study participants had a comprehensive ocular evaluation. We performed WES for 16 samples (nine POAG and seven unaffected controls) since Sanger sequencing of the POAG candidate genes (MYOC, OPTN, and TBK1) showed no genetic variation. We used an in-house pipeline for prioritizing the pathogenic variants based on their segregation among the POAG individual. Results: We identified one novel and five low-frequency pathogenic variants with consistent co-segregation in all affected individuals. The variant c.G3719A in RPGR-interacting domain of RPGRIP1 that segregated heterozygously with the six POAG cases is distinct from variants causing photoreceptor dystrophies, reported affecting the RPGR protein complex signaling in primary cilia. The cilia in trabecular meshwork (TM) cells has been reported to mediate the intraocular pressure (IOP) sensation. Furthermore, we identified a novel c.A1295G variant in Rho guanine nucleotide exchange factors Gene 40 (ARHGEF40) and a likely pathogenic variant in the RPGR gene, suggesting that they may alter the RhoA activity essential for IOP regulation. Conclusion: Our study supports that low-frequency pathogenic variants in multiple genes and pathways probably affect Primary Open Angle Glaucoma's pathogenesis in the large South Indian family. Furthermore, it requires larger case-controls to perform family-based association tests and to strengthen our analysis.</p>
          </details>
        </div>

        <!-- Paper 8 - 2020 -->
        <div class="publication-card">
          <h4><a href="https://pubmed.ncbi.nlm.nih.gov/31934596/" target="_blank">
              Genetic characterization of Stargardt clinical cases
          </a></h4>
          <p><strong>Raj, Rajendran Kadarkarai, Manojkumar Kumaran et al.</strong> <i>Eye and Vision</i> (2020)</p>

          <details>
              <summary>Show Abstract</summary>
              <p>

                Background: Stargardt disease 1 (STGD1; MIM 248200) is a monogenic form of autosomal recessive genetic disease caused by mutation in ABCA4. This gene has a major role in hydrolyzing N-retinylidene-phosphatidylethanolamine to all-trans-retinal and phosphatidylethanolamine. The purpose of this study is to identify the frequency of putative disease-causing mutations associated with Stargardt disease in a South Indian population.
              </p> <p>  
                Methods: A total of 28 clinically diagnosed Stargardt-like phenotype patients were recruited from south India. Ophthalmic examination of all patients was carefully carried out by a retina specialist based on the stages of fundus imaging and ERG grouping. Genetic analysis of ABCA4 was performed for all patients using Sanger sequencing and clinical exome sequencing.
              </p> <p> 
                Results: This study identified disease-causing mutations in ABCA4 in 75% (21/28) of patients, 7% (2/28) exhibited benign variants and 18% (5/28) were negative for the disease-causing mutation.
              </p> <p>
                Conclusion: This is the first study describing the genetic association of ABCA4 disease-causing mutation in South Indian Stargardt 1 patients (STGD1). Our findings highlighted the presence of two novel missense mutations and an (in/del, single base pair deletion & splice variant) in ABCA4. However, genetic heterogeneity in ABCA4 mutants requires a larger sample size to establish a true correlation with clinical phenotype.
                </p>
          </details>
      </div>

        <!-- Paper 9 - 2019 -->
        <div class="publication-card">
            <h4><a href="https://pubmed.ncbi.nlm.nih.gov/31208315/" target="_blank">
                Performance assessment of variant calling pipeline using human whole exome sequencing and simulated data
            </a></h4>
            <p><strong>Manojkumar Kumaran et al.</strong> <i>BMC Bioinformatics</i> (2019)</p>

            <details>
                <summary>Show Abstract</summary>
                <p>

                  Background: Whole exome sequencing (WES) is a cost-effective method that identifies clinical variants but it demands accurate variant caller tools. Currently available tools have variable accuracy in predicting specific clinical variants. But it may be possible to find the best combination of aligner-variant caller tools for detecting accurate single nucleotide variants (SNVs) and small insertion and deletion (InDels) separately. Moreover, many important aspects of InDel detection are overlooked while comparing the performance of tools, particularly its base pair length.
                </p> <p>
                  Results: We assessed the performance of variant calling pipelines using the combinations of four variant callers and five aligners on human NA12878 and simulated exome data. We used high confidence variant calls from Genome in a Bottle (GiaB) consortium for validation, and GRCh37 and GRCh38 as the human reference genome. Based on the performance metrics, both BWA and Novoalign aligners performed better with DeepVariant and SAMtools callers for detecting SNVs, and with DeepVariant and GATK for InDels. Furthermore, we obtained similar results on human NA24385 and NA24631 exome data from GiaB.
                </p> <p>
                  Conclusion: In this study, DeepVariant with BWA and Novoalign performed best for detecting accurate SNVs and InDels. The accuracy of variant calling was improved by merging the top performing pipelines. The results of our study provide useful recommendations for analysis of WES data in clinical genomics.
                  </p>
            </details>
        </div>
       </div>
</div>
          
           
          </div>
        </div>
      </section>
      <!-- publication Section End -->

      <!-- Portfolio Section -->
<section class="portfolio section" id="portfolio">
  <div class="container">
    <div class="row">
      <div class="section-title padd-15">
        <h2>Services</h2>
      </div>
    </div>
    <div class="row">
      <div class="portfolio-filter padd-15">
        <button type="button" class="active" data-filter="all">All</button>
        <button type="button" data-filter="web-design">Web Design</button>
        <button type="button" data-filter="photography">Photography</button>
      </div>
    </div>
    <div class="row">
      <!-- Portfolio Item 1 -->
      <div class="portfolio-item padd-15" data-category="web-design">
        <div class="portfolio-item-inner shadow-dark">
          <a href="https://bdu-biodiversity.blogspot.com/p/blog-page_5459.html" target="_blank">
            <div class="portfolio-img">
              <img src="images/portfolio/1.jpg" alt="Biodiversity Blog">
            </div>
            <div class="portfolio-info">
              <h4>I have created the Bharathidasan University Biodiversity Blog to document and showcase the rich biodiversity of our campus This platform aims to connect nature enthusiasts, researchers, and students in sharing photographs, species identification, and conservation efforts.</h4>
              <div class="icon">
                <i class="fa fa-external-link"></i>
              </div>
            </div>
          </a>
        </div>
      </div>
      <!-- Portfolio Item End -->

      <!-- Portfolio Item 2 -->
      <div class="portfolio-item padd-15" data-category="web-design">
        <div class="portfolio-item-inner shadow-dark">
          <a href="https://venkateshlab.science/" target="_blank">
            <div class="portfolio-img">
              <img src="images/portfolio/2.jpg" alt="Venkatesh Lab">
            </div>
            <div class="portfolio-info">
              <h4>Venkatesh Lab is dedicated to advancing research in Neuroscience. Our work focuses on understanding the axon growth regulated during development and regeneration in mammals, fostering collaboration among scientists and students. As the website admin, I ensure regular updates, making it a hub for research insights and discoveries.</h4>
              <div class="icon">
                <i class="fa fa-external-link"></i>
              </div>
            </div>
          </a>
        </div>
      </div>
      <!-- Portfolio Item End -->
    </div>
  </div>
</section>
<!-- Portfolio Section End -->

      <!-- Blog Section -->
      <section class="blog section" id="blog">
        <div class="container">
          <div class="row">
            <div class="section-title padd-15">
              <h2>Latest Blog</h2>
            </div>
          </div>
          <div class="row">
            <!-- Blog Item -->
            <div class="blog-item padd-15">
              <div class="blog-item-inner shadow-dark">
                <div class="blog-img">
                  <img src="images/blog/1.jpg" alt="">
                  <div class="blog-date">
                    3 jan 2025
                  </div>
                </div>
                <div class="blog-info">
                  <h4 class="blog-title">Bee Eater</h4>
                  <p class="blog-description">The quiet beauty of nature lies in its simplicity, where every flutter of a wing and every whisper of a breeze tells a story of life unfolding..</p>
                  <p class="blog-tags">Tags : <a href="#">Wildlife</a> , <a href="#">Photography</a></p>
                </div>
              </div>
            </div>
            <!-- Blog Item End -->
            <!-- Blog Item -->
            <div class="blog-item padd-15">
              <div class="blog-item-inner shadow-dark">
                <div class="blog-img">
                  <img src="images/blog/2.jpg" alt="">
                  <div class="blog-date">
                    25 Dec 2024
                  </div>
                </div>
                <div class="blog-info">
                  <h4 class="blog-title">Drops of Serenity</h4>
                  <p class="blog-description">A single drop holds the universe, reflecting the beauty of nature in its purest form. Just like life, every moment is fleeting yet infinitely profound</p>
                  <p class="blog-tags">Tags : <a href="#">Photography</a></p>
                </div>
              </div>
            </div>
            <!-- Blog Item End -->
            <!-- Blog Item -->
            <div class="blog-item padd-15">
              <div class="blog-item-inner shadow-dark">
                <div class="blog-img">
                  <img src="images/blog/3.jpg" alt="">
                  <div class="blog-date">
                    14 Dec 2024
                  </div>
                </div>
                <div class="blog-info">
                  <h4 class="blog-title">Unraveling Neural Pathways</h4>
                  <p class="blog-description">LThe brain's complexity is reflected in its genomic blueprint—each gene, a note; each pathway, a melody shaping the orchestra of neural development.</p>
                  <p class="blog-tags">Tags : <a href="#">Neuroscience</a>,<a href="#">Bioinformatics </a> </p>
                </div>
              </div>
            </div>
            <!-- Blog Item End -->
          </div>
        </div>
      </section>
      <!-- Blog Section End -->

      <!-- Testimonial Section -->
<section class="testimonial section" id="testimonial">
  <div class="container">
    <div class="row">
      <div class="section-title padd-15">
        <h2>Testimonials</h2>
      </div>
    </div>
    <div class="row">
      <!-- Testimonial Item 1 -->
      <div class="testimonial-item padd-15">
        <div class="testimonial-item-inner shadow-dark">
          <div class="testimonial-img">
            <img src="images/testimonial1.jpg" alt="Dr. John Doe">
          </div>
          <p>"Manoj is an exceptional bioinformatician. His expertise in genomics and software development is truly inspiring!"</p>
          <h4>- Dr. Ishwariya Venkatesh</h4>
        </div>
      </div>
      <!-- Testimonial Item 2 -->
      <div class="testimonial-item padd-15">
        <div class="testimonial-item-inner shadow-dark">
          <div class="testimonial-img">
            <img src="images/testimonial2.jpg" alt="Dr. Jane Smith">
          </div>
          <p>"Working with Manoj has been an absolute pleasure. His ability to automate complex workflows is unmatched."</p>
          <h4>- Dr. Bharanidharan</h4>
        </div>
      </div>
      <!-- Testimonial Item 3 -->
      <div class="testimonial-item padd-15">
        <div class="testimonial-item-inner shadow-dark">
          <div class="testimonial-img">
            <img src="images/testimonial3.jpg" alt="Prof. Richard Lee">
          </div>
          <p>"A brilliant researcher with an innovative approach to computational biology. Highly recommended!"</p>
          <h4>- Vatsal Mehara</h4>
        </div>
      </div>
    </div>
  </div>
</section>



      <!-- Contact Section -->
      <section class="contact section" id="contact">
        <div class="container">
          <div class="row">
            <div class="section-title padd-15">
              <h2>Contact Me</h2>
            </div>
          </div>
          <div class="row">
            <!-- contact-info-item  -->
            <div class="contact-info-item padd-15">
              <div class="icon">
                <i class="fa fa-phone"></i>
                <h4>Call Me On</h4>
                <p>+91 7708556754</p>
              </div>
            </div>
            <!-- contact-info-item end -->
            <!-- contact-info-item  -->
            <div class="contact-info-item padd-15">
              <div class="icon">
                <i class="fa fa-map-marker"></i>
                <h4>Office</h4>
                <p>Centre for Cellular & Molecular Biology,
                  Habsiguda, Uppal,
                  Hyderabad - 500 007</p>
              </div>
            </div>
            <!-- contact-info-item end -->
            <!-- contact-info-item  -->
            <div class="contact-info-item padd-15">
              <div class="icon">
                <i class="fa fa-envelope"></i>
                <h4>Email</h4>
                <p>manojkumarbioinfo@gmail.com</p>
              </div>
            </div>
            <!-- contact-info-item end -->
          </div>

          <!-- Contact Form -->
          <div class="row">
            <form action="#" class="contact-form padd-15">
              <div class="row">
                <div class="form-item col-6 padd-15">
                  <div class="form-group">
                    <input type="text" name="" class="form-control" placeholder="Name*">
                  </div>
                </div>
                <div class="form-item col-6 padd-15">
                  <div class="form-group">
                    <input type="email" name="" class="form-control" placeholder="Email*">
                  </div>
                </div>
              </div>
              <div class="row">
                <div class="form-item col-12 padd-15">
                  <div class="form-group">
                    <input type="text" name="" class="form-control" placeholder="Subject*">
                  </div>
                </div>
              </div>
              <div class="row">
                <div class="form-item col-12 padd-15">
                  <div class="form-group">
                    <textarea class="form-control" name="" placeholder="Your Message..."></textarea>
                  </div>
                </div>
              </div>
              <div class="row">
                <div class="col-12 padd-15">
                  <button type="submit" class="btn">Send Message</button>
                </div>
              </div>
            </form>
          </div>
          <!-- Contact Form End -->
        </div>
      </section>
      <!-- Contact Section End -->

    </div>
    <!-- Main Content End -->
  </div>
  <!-- Main Container End -->

  <!-- Lightbox -->
  <div class="lightbox">
    <div class="lightbox-content">
      <div class="lightbox-close">&times;</div>
      <img src="" alt="" onclick="nextItem()" class="lightbox-img">
      <div class="lightbox-caption">
        <div class="caption-text"></div>
        <div class="caption-counter"></div>
      </div>
    </div>
    <div class="lightbox-controls">
      <div class="prev-item" onclick="prevItem()"><i class="fa fa-angle-left"></i></div>
      <div class="next-item" onclick="nextItem()"><i class="fa fa-angle-right"></i></div>
    </div>
  </div>
  <!-- Lightbox End -->

  <!-- Live Style Switcher - Demo Only -->
  <div class="style-switcher">
    <div class="toggle-style-switcher">
      <i class="fa fa-cog fa-spin"></i>
    </div>
    <h5>Style Switcher</h5>
    <ul>
      <li><a href="javascript:void(0);" title="pink" style="background-color: #ec1839;" onclick="setActiveStyle('pink')"></a></li>
      <li><a href="javascript:void(0);" title="blue" style="background-color: #2196f3;" onclick="setActiveStyle('blue')"></a></li>
      <li><a href="javascript:void(0);" title="orange" style="background-color: #fa5b0f;" onclick="setActiveStyle('orange')"></a></li>
      <li><a href="javascript:void(0);" title="yellow" style="background-color: #ffb400;" onclick="setActiveStyle('yellow')"></a></li>
      <li><a href="javascript:void(0);" title="green" style="background-color: #72b626;" onclick="setActiveStyle('green')"></a></li>
    </ul>
    <h5>Body Skin</h5>
    <label>
      <input type="radio" class="body-skin" name="body-skin" value="light" checked="true" > Light
    </label>
    <label>
      <input type="radio" class="body-skin" name="body-skin" value="dark" > Dark
    </label>
  </div>
  <!-- Live Style Switcher - Demo Only End -->

  <!-- Plugins -->
  <script src="js/ityped.min.js"></script>
  <!-- Template js -->
  <script src="js/script.js"></script>
  <!-- Live Style Js - Demo Only -->
  <script src="js/styleSwitcher.js"></script>
</body>

</html>
